components/brave_rewards/browser/rewards_service.h

```diff
@@ -176,6 +181,29 @@ class RewardsService : public KeyedService {
   virtual void GetConfirmationsHistory(
       brave_rewards::ConfirmationsHistoryCallback callback) = 0;
 
+  virtual void GetAutoContributeCount(
+      const rewards_counter::OnRewardsCountedCallback) = 0;
+
+  virtual void RemoveAutoContributeData(
+      rewards_data::OnRemoveAutoContributeDataCallback callback) = 0;
+
+  virtual void ClearAutoContributeData(
+      rewards_data::OnRemoveAutoContributeDataCallback callback,
+      uint64_t reconcile_stamp) = 0;
+
+  virtual void OnClearAutoContributeData(
+      rewards_data::OnRemoveAutoContributeDataCallback callback,
+      bool result) = 0;
+
+  virtual void RemoveAllRewardsData(
+      rewards_data::OnRemoveAllRewardsDataCallback callback) = 0;
+
+  virtual void OnRemoveAllRewardsData(
```

Comment:
Can this one be moved into impl as it's callback function

Reply:
They have been removed. They were callbacks that ran after each clearing task. There is already one callback that occurs after all clearing tasks are completed.